tests/scanners/test.langpack.js

```diff
@@ -4,61 +4,52 @@ import PropertiesParser from 'parsers/properties';
 import DoctypeParser from 'parsers/doctype';
 import FluentParser from 'parsers/fluent';
 
-import { unexpectedSuccess } from '../helpers';
-
 
 describe('LangpackScanner', () => {
   it('should report a proper scanner name', () => {
     expect(LangpackScanner.scannerName).toEqual('langpack');
   });
 
-  it('scan property files', () => {
-    const spy = sinon.spy(PropertiesParser.prototype, 'parse');
+  it('should scan property files', async () => {
     const code = 'foo = bar';
-    const scanner = new LangpackScanner(code, 'foo.properties');
+    const langpackScanner = new LangpackScanner(code, 'foo.properties');
+
+    sinon.spy(PropertiesParser.prototype, 'parse');
 
-    return scanner.scan()
-      .then(({ linterMessages }) => {
-        expect(linterMessages.length).toEqual(0);
-        sinon.assert.calledOnce(spy);
-      });
+    const { linterMessages } = await langpackScanner.scan();
+    expect(linterMessages.length).toEqual(0);
+    sinon.assert.calledOnce(PropertiesParser.prototype.parse);
   });
 
-  it('scan dtd files', () => {
-    const spy = sinon.spy(DoctypeParser.prototype, 'parse');
+  it('should scan dtd files', async () => {
     const code = '<!ENTITY foo "bar">';
-    const scanner = new LangpackScanner(code, 'foo.dtd');
+    const langpackScanner = new LangpackScanner(code, 'foo.dtd');
 
-    return scanner.scan()
-      .then(({ linterMessages }) => {
-        expect(linterMessages.length).toEqual(0);
-        sinon.assert.calledOnce(spy);
-      });
+    sinon.spy(DoctypeParser.prototype, 'parse');
```

Comment:
Same here: let's configure the sinon spy on DoctypeParser.prototype before the line with `... = new LangpackScanner(...)`.